internal/extsvc/gerrit/client.go

```diff
@@ -166,8 +166,10 @@ func (c *client) do(ctx context.Context, req *http.Request, result any) (*http.R
 				Body:       bs,
 			}
 		}
-
-		bs = bs[4:]
+		// Gerrit attaches this prefix to most of its responses, so if it exists, we cut it, so we can parse it as a json properly.
+		if len(bs) >= 4 && string(bs[:4]) == ")]}'" {
```

Comment:
You also don't need the cast:

```go
		if len(bs) >= 4 && bs[:4] == ")]}'" {
```

Or, more idiomatic:

```suggestion
		if strings.HasPrefix(s, ")]}'") {
```